### PR TITLE
uhm-7593-Migrate ZL MCH non-coded to coded diagnoses

### PR DESCRIPTION
**Pull request by dmdesimone**

### CODE REVIEW

**Review by lnball — APPROVED** [review]
LGTM

File: configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/*
+ * This is a one-time script to migrate several diagnoses entered
+ * as non-coded int he past that should now be coded
+ */
+
+-- set up translation table for diagnoses
+drop table if exists dx_translations;
```
Comment (mseaton) [inline configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql:7]: can this be changed to `drop temporary table...` just to be safe?

File: configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+	('PES prob.',concept_from_mapping('CIEL','113006')),
+	('AGUS',concept_from_mapping('CIEL','155209')),
+	('PCOS',concept_from_mapping('CIEL','129569')),
+	('HTA non controlee',concept_from_mapping('CIEL','165587')),
+	('anamnios',concept_from_mapping('CIEL','168588')),
+	('BDCF non rassurant',concept_from_mapping('CIEL','168595')),
+	('uc2 ancien',concept_from_mapping('CIEL','168594')),
+	('anemie probable',concept_from_mapping('CIEL','121629')),
+	('douleur epigastrique',concept_from_mapping('CIEL','141128')),
+	('PES stab',concept_from_mapping('CIEL','168611'))
```
Comment (mseaton) [inline configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql:62]: I did not review any of these mappings for accuracy...

File: configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+	concept_id,
+	encounter_id,
+	obs_datetime,
+	location_id,
+	obs_group_id,
+	creator,
+	date_created,
+	status,
+	value_coded,
+	uuid)
```
Comment (mseaton) [inline configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql:136]: Just a note to make sure you sanity check that there are no other columns with meaningful data that you need to preserve.

File: configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+	
+-- insert coded obs
+insert into obs (
+	person_id,
+	concept_id,
+	encounter_id,
+	obs_datetime,
+	location_id,
+	obs_group_id,
+	creator,
```
Comment (mseaton) [inline configuration/pih/liquibase/sql/migrate_non_coded_mch_dxs.sql:132]: One thing are you missing here is that you should probably set the `previous_version` of the new obs to the obs_id of the old obs that is being voided.

**Review by mseaton — COMMENTED** [review]
LGTM generally @dmdesimone .  A few comments, but nothing major.

**Pull request merged**